DOCUMENTATION.md

```diff
@@ -158,6 +158,17 @@ personalisation={
 }
 ```
 
+If your service has a permission to upload documents you can pass file objects as values for the personalisation keys:
+
+```python
+with open('file.pdf', 'rb') as f:
+    ...
+    personalisation={
+      'first_name': 'Amala',
+      'document': f,
```

Comment:
dont know if we want to document this process more at the client lib level to indicate that this magic file pointer thing will be a link in the output. probably not tho

Reply:
I think people will have questions, but doing the minimal thing for now lets us find out what those questions are in research 👍

Reply:
Had a chat with @richardc0 and @allait on this. We'll coordinate with Tom Chambers to make sure that the information about the feature is on the features page. I have a couple of questions about this and would like to answer those before this gets merged.